modules/nixvim/nixvim.nix

```diff
@@ -22,6 +22,26 @@ in
       main = lib.mkEnableOption "background transparency for the main NeoVim window";
       signColumn = lib.mkEnableOption "background transparency for the NeoVim sign column";
     };
+    config = lib.mkOption {
```

Comment:
In #415, the consensus _seemed_ to be that we should assign the nixvim config to `config.lib.stylix.nixvim.config`.

Doing that is incompatible with declaring an option, though. So it would be more difficult to document.